<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QStackedWidget" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>2</number>
  </property>
  <widget class="QWidget" name="loading">
   <widget class="QLabel" name="loading_2">
    <property name="geometry">
     <rect>
      <x>814</x>
      <y>604</y>
      <width>206</width>
      <height>116</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="loadingback">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>720</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/loading/background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>loadingback</zorder>
   <zorder>loading_2</zorder>
  </widget>
  <widget class="QWidget" name="menu">
   <widget class="QLabel" name="player_data">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>210</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/menu/player_data.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player_data_0">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>430</y>
      <width>180</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/title/player_data_0.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="quit_0">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>500</y>
      <width>180</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/title/quit_0.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="rank_select">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>210</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/menu/rank_select.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="game_start_0">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>180</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/title/game_start_0.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="hard">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>340</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/menu/hard.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="normal">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>340</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/menu/normal.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>720</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/menu/background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="game_start_1">
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>360</y>
      <width>180</width>
      <height>39</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/title/game_start_1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lunatic">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>340</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/menu/lunatic.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>678</width>
      <height>167</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/title/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player_data_1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>430</y>
      <width>180</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/title/player_data_1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="quit_1">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>500</y>
      <width>180</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/title/quit_1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="author">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>670</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Made by 蒼崎ゆうま</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="easy">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>340</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/menu/easy.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="score_0">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="score_1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="score_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>420</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>player_data</zorder>
   <zorder>player_data_0</zorder>
   <zorder>quit_0</zorder>
   <zorder>rank_select</zorder>
   <zorder>game_start_0</zorder>
   <zorder>hard</zorder>
   <zorder>normal</zorder>
   <zorder>game_start_1</zorder>
   <zorder>lunatic</zorder>
   <zorder>title</zorder>
   <zorder>player_data_1</zorder>
   <zorder>quit_1</zorder>
   <zorder>author</zorder>
   <zorder>easy</zorder>
   <zorder>score_0</zorder>
   <zorder>score_1</zorder>
   <zorder>score_2</zorder>
  </widget>
  <widget class="QWidget" name="game">
   <widget class="QLabel" name="background_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>720</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/gamebg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="rank_e">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/easy.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="rank_n">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/normal.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="rank_h">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/hard.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="rank_l">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/lunatic.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bomb_0">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>222</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/bomb.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bomb_1">
    <property name="geometry">
     <rect>
      <x>785</x>
      <y>222</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/bomb.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="life_0">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>177</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/life.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="life_1">
    <property name="geometry">
     <rect>
      <x>785</x>
      <y>177</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/life.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mask">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>24</y>
      <width>615</width>
      <height>672</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/pause_back.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="game_pause">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/game_pause.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="game_over">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/game_over.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="continue_0">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>510</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/continue_0.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="continue_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>510</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/continue_1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="restart_0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>550</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/restart_0.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="restart_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>550</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/restart_1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="return_title_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>590</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/return2title_0.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="return_title_1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>590</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/front/return2title_1.png</pixmap>
    </property>
   </widget>
   <widget class="QLCDNumber" name="high_score">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>80</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>-1</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="score">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>120</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>-1</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLabel" name="battlefeild">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>24</y>
      <width>615</width>
      <height>672</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/background/BluePinkNebular_top.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="player_name">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color: rgb(85, 85, 255);
border-width:0;
border-style:outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="high_score_plain">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color: rgb(255, 0, 255);
border-width:0;
border-style:outset;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>dalao你打破记录辣
    快留下你的美名
     (最多12个字符)</string>
    </property>
    <property name="backgroundVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="yes">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border-width:0;
border-style:outset;</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/front/ok.png</normaloff>:/res/front/ok.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <zorder>background_2</zorder>
   <zorder>battlefeild</zorder>
   <zorder>mask</zorder>
   <zorder>rank_e</zorder>
   <zorder>rank_n</zorder>
   <zorder>rank_h</zorder>
   <zorder>rank_l</zorder>
   <zorder>bomb_0</zorder>
   <zorder>bomb_1</zorder>
   <zorder>life_0</zorder>
   <zorder>life_1</zorder>
   <zorder>game_pause</zorder>
   <zorder>game_over</zorder>
   <zorder>continue_0</zorder>
   <zorder>continue_1</zorder>
   <zorder>restart_0</zorder>
   <zorder>restart_1</zorder>
   <zorder>return_title_0</zorder>
   <zorder>return_title_1</zorder>
   <zorder>high_score</zorder>
   <zorder>score</zorder>
   <zorder>player_name</zorder>
   <zorder>high_score_plain</zorder>
   <zorder>yes</zorder>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
